src/Nethermind/Nethermind.Core/Crypto/Keccak.cs

```diff
@@ -48,6 +51,17 @@ public static ValueHash256 Compute(string input)
             return InternalCompute(System.Text.Encoding.UTF8.GetBytes(input));
         }
 
+        private readonly struct BytesWrapper(byte[] bytes) : IEquatable<BytesWrapper>
+        {
+            internal readonly byte[] _bytes = bytes;
+            public static implicit operator BytesWrapper(byte[] bytes) => new(bytes);
+            public static implicit operator BytesWrapper(ReadOnlySpan<byte> bytes) => new(bytes.ToArray());
```

Comment:
Can we search by Span without allocating first, like in SpanDictionary?

Reply:
Not in the regular ConcurrentDictionary; though it should live an die pretty fast in Gen0.

Also another reason to limiting to 32 bytes

Reply:
Well it is good I made SpanConcurrentDictionary ]:->